project/LibraryVersions.scala

```diff
@@ -2,8 +2,9 @@
 object LibraryVersions {
   val circeVersion = "0.12.1"
   val awsClientVersion = "1.11.827"
-  val catsVersion = "1.4.0"
-  val jacksonVersion = "2.12.0"
+  val catsVersion = "2.1.1"
```

Comment:
if we go any further on the cats version, it pulls in a simulacrum library that has an assembly merge conflict, which I wasn't sure how to resolve, as one is a scalafix thing and the other is a real typeclass implementation.